src/plays/age-calculator-by-react/styles.css

```diff
@@ -0,0 +1,330 @@
+/* enter stlyes here */
```

Comment:
Add a unique style name infront of every style defined here... otherwise it will override things.

Reply:
Done